contracts/src/PBHSignatureAggregator.sol

```diff
@@ -32,13 +32,6 @@ contract PBHSignatureAggregator is IAggregator {
     ///                                  ERRORS                                ///
     //////////////////////////////////////////////////////////////////////////////
```

Comment:
Looks like these changes are related to `N-06`. Can we remove these changes in favor of [#145?](https://github.com/worldcoin/world-chain/pull/145)